.github/workflows/cnb.yml

```diff
@@ -0,0 +1,99 @@
+name: CNB
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 4 * * 1-5'
+
+defaults:
+  run:
+    # Setting an explicit bash shell ensures GitHub Actions enables pipefail mode too,
+    # rather than only error on exit (improving failure UX when pipes are used). See:
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
+    shell: bash
+
+permissions:
+  actions: write
+  contents: write
+  pull-requests: write
```

Comment:
These permissions aren't used when using the GitHub App, so can be disabled similar to:
https://github.com/heroku/heroku-buildpack-python/blob/07005f2a3f56c78d9ae51398378e7ae629aa1f54/.github/workflows/prepare-release.yml#L6-L7